apps/antalmanac/src/components/RightPane/CoursePane/CourseRenderPane.tsx

```diff
@@ -230,17 +231,24 @@ class CourseRenderPane extends PureComponent<CourseRenderPaneProps, CourseRender
                 division: formData.division,
             };
 
+            const gradesQueryParams = {
+                department: formData.deptValue,
+                ge: formData.ge as GE,
+            };
+
             try {
-                let jsonResp;
-                if (params.units.includes(',')) {
-                    jsonResp = await queryWebsocMultiple(params, 'units');
-                } else {
-                    jsonResp = await queryWebsoc(params);
-                }
+                // Query websoc for course information and populate gradescache
+                const [websocJsonResp, _] = await Promise.all([
```

Comment:
I think it's fine to omit the dummy/placeholder variable.

```suggestion
                const [websocJsonResp] = await Promise.all([
```

Reply:
Huh. I didn't know you could do that. Python would not be happy if you did that.